metals/src/main/scala/scala/meta/internal/metals/debug/server/Logger.scala

```diff
@@ -22,6 +30,20 @@ class Logger(listener: DebuggeeListener) {
         val address = new InetSocketAddress("127.0.0.1", port)
         listener.onListening(address)
       }
-    } else listener.out(msg)
+    } else if (
+      specialHandleStacktrace && StackTraceMatcher.isStackTraceLine(msg)
+    ) listener.err(msg)
+    else listener.out(msg)
```

Comment:
We could instead add the location in https://github.com/scalameta/metals/blob/a588c7b0f066f2df020f2525ac57d60835f02fbc/metals/src/main/scala/scala/meta/internal/metals/debug/DebugProxy.scala#L276 similar to error and it should also work for Bloop/sbt

I would use StackTraceMatcher.isStackTraceLine(msg) though since that's a quicker way of figuring out if something is a stack trace. Getting a symbol actually my slow down printing.

`if (output.getOutput().contains("at "))` from error output could be replaced